scripts/setup.sh

```diff
@@ -177,6 +198,7 @@ check_if_parameter_is_empty "$GCP_IAM_ROLE" "'googleCloud.common.iamRole'"
 check_if_parameter_is_empty "$GCP_SERVICE_ACCOUNT" "'googleCloud.common.serviceAccount'"
 check_if_parameter_is_empty "$GCP_SCHEDULER_CRON" "'googleCloud.metrics.schedulerSchedule'"
 
+echo  "Logging to your account..."
```

Comment:
minor comments - lets keep messages format consistent, you can add '- ' at the begonning